pkg/apis/migration/v1alpha1/directvolumemigration_types.go

```diff
@@ -88,6 +88,23 @@ func (ds *DirectVolumeMigrationStatus) GetRsyncOperationStatusForPVC(pvcRef *kap
 	return newStatus
 }
 
+// AddRsyncOperation adds a new RsyncOperation to list, updates an existing one if found
+func (ds *DirectVolumeMigrationStatus) AddRsyncOperation(podStatus *RsyncOperation) {
+	if podStatus == nil {
```

Comment:
out of curiosity do you need to remove the rsync operation from the list here if found?

Reply:
actually nevermind, I was misreading.